scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram_transducer.py

```diff
@@ -0,0 +1,264 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script would evaluate an N-gram language model trained with KenLM library (https://github.com/kpu/kenlm) in
```

Comment:
Docs here needs update.

Reply:
I am going to merge `eval_beamsearch_ngram_transducer.py` with `eval_beamsearch_ngram.py` in another PR. Could we leave `eval_beamsearch_ngram_transducer.py` as is now? I will update docs according to new `eval_beamsearch_ngram.py`.